modules/cluster/local.tf

```diff
@@ -0,0 +1,18 @@
+provider "random" {
+  version = "~> 2.1"
+}
+
+resource "random_string" "suffix" {
+  length  = 8
+  special = false
+}
+
+// ----------------------------------------------------------------------------
+// Module local variables
+// ----------------------------------------------------------------------------
+locals {
+  generated_seed         = random_string.suffix.result
+  oidc_provider_url      = replace(module.eks.cluster_oidc_issuer_url, "https://", "")
+  jenkins-x-namespace    = "jx"
```

Comment:
why would you consider the jenkins-x and cert-manager namespaces as local? Are they not already defaulted in the definition of the variables?

Reply:
Removed them, I'm not even using them anymore.